src/main/java/salesforce/ui/pages/opportunity/CreatedOpportunityPage.java

```diff
@@ -0,0 +1,152 @@
+
+    @FindBy(xpath = "//a[@data-tab-value='detailTab']")
+    private WebElement detailBtn;
+
+    private By menuBtn = By.xpath("//li[contains(@class,'slds-dropdown-trigger_click')]");
+    private By deleteOption = By.xpath("//div[@role='menu']//*[@title='" + Translator.translateValue("Opportunity",
+            "delete") + "']//a");
+    private By deleteBtnConfirm = By.xpath("//button[@title=' + ConfigEnvVar.getResourceBundle()."
+            + "getString(\"delete\") + \"']");
+    private static final String CREATED_HEADER = "//div[./p[text()='%s']]//%s";
+    private static final String CREATED_DETAIL = "//div[./div[./span[contains(text(),'%s')]]]"
+            + "//lightning-formatted-text";
+    private static final String CREATED_SEARCH = "//div[./div[./span[contains(text(),'%s')]]]/"
+            + "div[@class='slds-form-element__control']";
+    private WebElementAction webElementAction = new WebElementAction();
+
+    /**
+     * Waits for an specific element to load on the page.
+     */
+    @Override
+    protected void waitForPageToLoad() {
+        getWait().until(ExpectedConditions.visibilityOf(detailBtn));
+    }
+
+    /**
+     * Gets the alert text when an opportunity is created correctly.
+     * @return the text of the alert when opportunity is created.
+     */
+    public String getSuccessfulAlert() {
+        return webElementAction.getTextOnWebElement(textCreatedSuccessfull);
+    }
+
+    /**
+     * Gets the Title of the created opportunity page.
+     * @return the current text of the title header.
+     */
+    public String getTitleHeader() {
+        return webElementAction.getTextOnWebElement(createdOpportunityTitle);
+    }
+
+    /**
+     * Gets the generic text from the header in a created opportunity page.
+     * @param field name of the field to get the String.
+     * @param headerType type of the element field.
+     * @return the current text of the specific field.
+     */
+    public String getHeaderString(final String field, final String headerType) {
+        try {
+            WebElement createdHeader = getDriver().findElement(By.xpath(String.format(CREATED_HEADER, field,
+                    headerType)));
+            return webElementAction.getTextOnWebElement(createdHeader);
+        } catch (NoSuchElementException e) {
+            return "";
+        }
+    }
+
+    /**
+     * Gets the Stage form the created Opportunity page.
+     * @return A string with the current text of the Stage.
+     */
+    public String getCurrentStage() {
+        return webElementAction.getTextOnWebElement(createdOpportunityStage);
+    }
+
+    /**
+     * Clicks the detail button for created Oppoprtunity.
+     */
+    public void clickDetails() {
+        webElementAction.clickOnWebElement(getDriver().findElement(By.xpath("//a[@data-tab-value='detailTab']")));
+        //NEEDED TO WAIT FOR JAVASCRIPT TO LOAD
+        try {
+            Thread.sleep(sleepTime);
+        } catch (InterruptedException ie) {
+
```

Comment:
You should do something with the exception

Reply:
Is a temporal solution, since there is a problem with the javascript details Opportunity page, Yessica said she would try to com with other solution.